sdk/tables/Azure.Data.Tables/test-resources.json

```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "String"
+        }
+    },
+    "variables": {
+        "mgmtApiVersion": "2019-04-01",
+        "location": "[resourceGroup().location]",
+        "primaryAccountName": "[concat(parameters('baseName'), 'prim')]",
+        "location": "[resourceGroup().location]",
+        "encryption": {
+            "services": {
+                "file": {
+                    "enabled": true
+                },
+                "blob": {
+                    "enabled": true
+                }
+            },
+            "keySource": "Microsoft.Storage"
+        },
+        "networkAcls": {
+            "bypass": "AzureServices",
+            "virtualNetworkRules": [],
+            "ipRules": [],
+            "defaultAction": "Allow"
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "apiVersion": "[variables('mgmtApiVersion')]",
+            "name": "[variables('primaryAccountName')]",
+            "location": "[variables('location')]",
+            "sku": {
+                "name": "Standard_RAGRS",
+                "tier": "Standard"
+            },
+            "kind": "StorageV2",
+            "properties": {
+                "networkAcls": "[variables('networkAcls')]",
+                "supportsHttpsTrafficOnly": true,
+                "encryption": "[variables('encryption')]",
+                "accessTier": "Cool"
+            }
+        }
+    ],
+    "outputs": {
+      "PRIMARY_STORAGE_ACCOUNT_NAME": {
```

Comment:
Should these be prefixed or is the script doing that automatically now?

Reply:
Script is not prefixing template outputs automatically yet but it's the plan.

Reply:
So having these not prefixed is fine.

Reply:
Not sure. I followed the example of what storage was doing and there was no prefix there.

Reply:
I think you are correct, I'll make these unique to TABLES_